src/main/java/org/spongepowered/api/data/DataRegistration.java

```diff
@@ -53,13 +53,29 @@
      */
     Class<T> getManipulatorClass();
 
+    /**
+     * Gets the implementing class of the {@link DataManipulator} for this
+     * registration.
+     *
+     * @return The manipulator class registered
+     */
+    Class<? extends T> getImplementationClass();
```

Comment:
Is the implementing class optional?

Reply:
No. It returns the API class if there isn't a separate implementing class.

Reply:
Then you should clarify that in the documentation, personally speaking it makes better sense.